libraries/ESP8266HTTPClient/src/ESP8266HTTPClient.h

```diff
@@ -204,8 +215,11 @@ class HTTPClient
     int writeToStreamDataBlock(Stream * stream, int len);
 
 
+#ifdef KEEP_PRESENT_API
     TransportTraitsPtr _transportTraits;
-    std::unique_ptr<WiFiClient> _tcp;
+    std::unique_ptr<WiFiClient> _tcpDeprecated;
+#endif
+    WiFiClient* _tcp;
```

Comment:
Suggest `_client` — this is not necessarily (only) TCP, it can also be providing TLS.

Reply:
Changing _tcp into _client